release-notes/release-notes/5.7-release-notes.md

```diff
@@ -164,57 +164,91 @@ To improve your Connections experience, we made various optimizations to our Pol
+
+#### Radio buttons for selection
+
+- Replaced drop-down menus with radio buttons for the following options:
+  - **Sync Strategy**
+  - **Source Schema Data Types**
+  - **Source Schema "Add Column"**
+
+#### Improved visibility
+
+- Expanded the width of source, destination, and connections drop-down menus to ensure visibility, even on screens with varying sizes.
+
+#### Streamlined file-based source fields
+
+- Streamlined the organization of file-based source fields for greater efficiency.
+
+#### Simplified options
+
+- Eliminated the following fields for a more focused interface:
+  - **Source > Cinchy Table > Model**
+  - **Info > Version**
+- The **API Response Format** field has been removed from the REST Source configuration. This change reflects that the only supported response format is JSON.
+
+#### Refined order of operations
+
+- Reorganized the process steps, moving the **"Permissions"** step within the **"Info"** tab.
+
+#### Clearer terminology
+
+- Adjusted terminology for clarity and consistency:
+  - Renamed **Sync Behaviour** tab to **Sync Actions**.
+  - Replaced **Parameters** with **Variables**.
+  - Changed **"Sync Pattern"** to **Sync Strategy** in the Sync Actions tab.
+  - Updated **Column Mappings** to **Mappings** in the Destination tab.
+  - Substituted **Access Token** with **API Key** in the Copper Source, aligning with Copper's documentation language.
+
+#### Enhanced guidance
+
+- Included descriptive explanations in various sections, such as Mapping, Schema, and Sync Behaviour, to provide comprehensive guidance during data sync configuration.
+
+#### Unified language
+
+- Standardized language used in file-based connectors across all Sources.
+
+#### Improved clarity
+
+- Added clarifying text throughout the interface for smoother navigation and configuration, fostering a more user-friendly experience.
+
+#### Organizational enhancements
+
+- Grouped Sources by type, distinguishing between Batch and Event categories.
+- Implemented alphabetical sorting for improved accessibility and ease of locating connections.
+
+#### Simplified Destination setup
+
+We've streamlined the destination setup process for data syncs. When selecting a Source other than Cinchy, the destination is now automatically set as Cinchy Table. This enhancement speeds up the creation of data syncs.
 
 #### Unique identifiers for saved connections
 
-We now support unique endpoints for all saved connections to make sharing and collaborating on connections easier. Connections now have a unique URL that you can share with other platform users and link them directly to the saved configuration.
+To facilitate sharing and collaboration on connections, we've introduced unique endpoints for all saved connections. Each connection now possesses a unique URL that can be shared with other platform users. This URL links directly to the saved configuration.
```

Comment:
unique _URLs_

Calling them endpoints may lead people to think it's related to APIs